core/logic.py

```diff
@@ -1,13 +1,15 @@
 from time import time
-from core import locale_manager
 from saver import open_global_pref
 from schedule.exceptions import NoSchedule
 from schedule.api import get_schedule
 from schedule.prettify import prettify_schedule
 from schedule.tools import get_today_schedule, get_tomorrow_schedule
 from model.chat import Chat
 from replyer import BaseReply
+from json import load
```

Comment:
Please, move everything related to localization to `locale_manager`. We should use a separate module to control localization